<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportModuleForm</class>
 <widget class="QWidget" name="ImportModuleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="nameEdit">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Module name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pathEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Path to source file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="openButton">
     <property name="text">
      <string>Open source file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="importButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
